.drone/scripts/mentors.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -e
+
+if [[ "${DRONE_COMMIT_BRANCH}" != "stable" ]]; then
+  echo "Not running stable branch. Abort."
+  exit 0
+fi
```

Comment:
You should be able to just make this step not even happen in the pipeline with something like this in the Drone CI config in your new `publish-mentors` section:

```jsonnet
when: {branch: ["stable"]}
```